braindecode/datautil/serialization.py

```diff
@@ -6,17 +6,129 @@
 #
 # License: BSD (3-clause)
 
-import json
 import os
-from glob import glob
+import json
 import warnings
+from glob import glob
 
 import mne
+from mne.io.fiff import raw
 import pandas as pd
 
 from ..datasets.base import BaseDataset, BaseConcatDataset, WindowsDataset
 
 
+def create_fname_id_from_description(description):
+    # XXX: Enforce 'subject', 'recording' and/or 'run' in description?
```

Comment:
I think currently we can't assume existence os 'recording', 'run' or 'session', since MOABB is inconsistent about this naming. I created a related issue here: https://github.com/NeuroTechX/moabb/issues/208

Reply:
Also, how would this work for custom datasets?